sdk/resourcemanager/Azure.ResourceManager/tests/Samples/Sample4_MultiTenant.cs

```diff
@@ -0,0 +1,37 @@
+﻿#region Snippet:MultiTenant_Namespaces
+using System;
+using System.Threading.Tasks;
+using Azure.Core;
+using Azure.Identity;
+#endregion
+using NUnit.Framework;
+
+namespace Azure.ResourceManager.Tests.Samples
+{
+    public class Sample4_MultiTenant
+    {
+        [Test]
+        [Ignore("Only verifying that the sample builds")]
+        public async Task EnableCrossTenantAuthentication()
+        {
+            #region Snippet:Enable_Cross_Tenant_Authentication
+            string clientId = Environment.GetEnvironmentVariable("CLIENT_ID");
+            string clientSecret = Environment.GetEnvironmentVariable("CLIENT_SECRET");
+            string tenantId01 = Environment.GetEnvironmentVariable("TENANT_ID_01");
+            string tenantId02 = Environment.GetEnvironmentVariable("TENANT_ID_02");
+            string subscriptionId01 = Environment.GetEnvironmentVariable("SUBSCRIPTION_ID_01");
+
+            // Prepare client and policy for tenant01
+            ClientSecretCredential credForTenant01 = new ClientSecretCredential(tenantId01, clientId, clientSecret);
+            ClientSecretCredential credForTenant02 = new ClientSecretCredential(tenantId02, clientId, clientSecret);
+
+            string token = (await credForTenant02.GetTokenAsync(new Azure.Core.TokenRequestContext(
```

Comment:
Can we simplify this?
```suggestion
            string token = (await credForTenant02.GetTokenAsync(new TokenRequestContext(
```

Reply:
Fixed